dist/backdraft.js

```diff
@@ -1476,6 +1544,70 @@ _.extend(Plugin.factory, {
       this.trigger("draw", arguments);
     },
 
+    exportData : function(sUrl) {
+      var oSettings = this.dataTable.fnSettings;
+      var aoData = this.dataTable._fnAjaxParameters( oSettings );
+      this._addServerParams( aoData );
+      this._fetchCSV( sUrl, aoData );
+    },
+
+    _fetchCSV : function (sUrl, aoData) {
+      if (this.serverSideFiltering) {
+        var filterJson = {};
+        filterJson.name = "ext_filter_json";
+        filterJson.value = this._getFilteringSettings();
+        aoData.push(filterJson);
+      }
+      $.ajax({
+        url: sUrl,
+        data : aoData,
+        dataType : "text",
+        cache : false,
+        type : this.ajaxMethod || "GET",
+        success: function(response, status, xhr) {
```

Comment:
I dont think we want ajax here and all this work. At least for now I'd rather remove all this code and mirror / use our existing reporitng CSV download infrasctucture, which is to do a window.location to the same url but with .csv extension, and server responds with the disposition / attachment headers and browser does the right thing.

Rather not use a different method here for now.